app/scripts/services/serviceInstances.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .factory('ServiceInstancesService', function() {
+    return {
+      // NOTE: should this be a filter?  or will shared functionality grow here? 
```

Comment:
I think this should be a filter. Maybe more than one: `serviceClassDisplayName` and `serviceInstanceDisplayName`?